apps/src/lab2/levelEditors/validations/EditValidation.tsx

```diff
@@ -63,14 +65,20 @@ const EditValidation: React.FunctionComponent<EditValidationProps> = ({
         >
           <FontAwesome icon="arrow-up" title={undefined} className="icon" />
         </button>
-
         <button
           type="button"
           onClick={() => moveValidation(validation.key, 'down')}
           className={moduleStyles.moveValidationButton}
         >
           <FontAwesome icon="arrow-down" title={undefined} className="icon" />
         </button>
+        <button
```

Comment:
Noting for later - it would be great to go back and replace these all with DSCO buttons at some point since these were written pre-DSCO/FontAwesomeV6.

Reply:
Yep - this came up in the Design conversation linked above too.